src/index.js

```diff
@@ -1,24 +1,22 @@
-const express = require('express');
-const cors = require('cors');
+const dontenv = require('dotenv');
 const mongoose = require('mongoose');
+const app = require('./app');
 
-const router = require('./routes');
+dontenv.config();
 
-const app = express();
-const port = process.env.PORT || 4000;
+mongoose.connect(process.env.MONGO_DB_CONNECT_URL)
```

Comment:
this should be: 

```
mongoose.connect(
process.env.MONGO_DB_CONNECT_URL, 
{ maxPoolSize: process.env.MONGO_POOLSIZE || 1})
```

Reply:
they didn't explain that 😡

Reply:
it was on the file before you updated it.